awx_collection/test/awx/test_job_template.py

```diff
@@ -85,6 +85,7 @@ def test_job_launch_with_prompting(run_module, admin_user, project, inventory, m
     JobTemplate.objects.create(
         name='foo',
         project=project,
+        organization='Default',
```

Comment:
If you add `organization` as an argument for this test and then `organization=organization,` on this line, this test (and thus CI) should pass.